### PR TITLE
Make the example runnable

### PR DESCRIPTION
**Pull request by githubshrek**
The proposed changes, together with the style sheet suggestion proposed in the previous proposed change (and shown below), means that the example moves from being abstract, and students can directly copy/paste the example and see it work.

```
  <style>
  .on { background-color: yellow; }
  .off { background-color: black; color: white; }
  </style>
```

### CODE REVIEW

**Comment by arthurvr** [comment]
This PR highly overlaps with #534.

File: page/events/introduction-to-custom-events.md [hunk 1]
```diff
@@ -78,18 +78,18 @@ Let's make our example a little more interesting. We'll add another room to our
 
 ```
 <div class="room" id="kitchen">
-	<div class="lightbulb on"></div>
-	<div class="switch"></div>
-	<div class="switch"></div>
-	<div class="clapper"></div>
+	<div class="lightbulb on">Kitchen light</div>
+	<div class="switch">Kitchen switch 1</div>
+	<div class="switch">Kitchen switch 1</div>
```
Comment (arthurvr) [inline page/events/introduction-to-custom-events.md]: ping @githubshrek. This PR needs some updates.

**Comment by arthurvr** [comment]
@githubshrek Hey! We asked for the CLA multiple times and can't merge without a signature. I'm going to close this due to inactivity. It doesn't really make sense to keep old PRs open for really long. Hope you'll be able to come back to this and we can reopen.

**Comment by githubshrek** [comment]
I thought I already signed whatever you wanted. You tell me I have to sign something, but give me not a clue where I should go to sign it. I have no idea where to go.

**Comment by arthurvr** [comment]
@githubshrek You have a signature on our file but will need to include your real name in your git config. We told you that on all your other PRs as well. Anyways, glad you responded now :) Could you add your real name and update the PR with the things we commented?

**Comment by githubshrek** [comment]
My Git profile has had my real name ever since you asked for it. If you want me to put my real name elsewhere, please tell me where, and how I do that.

**Comment by arthurvr** [comment]
> My Git profile has had my real name ever since you asked for it.

We were talking about your actual git config (not your GitHub profile), which has `githubshrek` as name. Your full name will need to be included in there. Setting it would be as easy as:

```
$ git commit --amend --author="First Last <name@gmail.com>"
```

Run that on the branch you created for the PR and and fill in with your own details of course. These details should match the ones you signed the CLA with. After changing it you will need to force push your commit to your branch. 

```
$ git push <remote> <branch> -f
```

**Comment by arthurvr** [comment]
ping @githubshrek? It doesn't seem to have worked? If you have any issues, feel free to contact me on Freenode IRC as _arthurvr_. Would love to be able to land this change, it's been open for way too long.

**Comment by githubshrek** [comment]
Hi,

I have no idea how to do any of what you're talking about. I don't have a
git config. I don't even know what Freenode IRC is.

regards,


Hugh Prior

**Comment by agcolom** [comment]
@arthurvr This was discussed today with the legal team. We can land the PR and change the author's name using --author

**Pull request merged**